Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/PasswordVaultSamplePage.xaml.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Uno.Gallery.Views.Samples
+{
+	[SamplePage(SampleCategory.Features, "PasswordVault", Description = "The PasswordVault-API enables storing and retrieving of secret data in Uno applications.", DocumentationLink = "https://platform.uno/docs/articles/features/PasswordVault.html")]
```

Comment:
This should probably use the official WinRT API docs for the DocumentationLink (https://learn.microsoft.com/en-us/uwp/api/windows.security.credentials.passwordvault) as well as the official Description: "Represents a Credential Locker of credentials. Lockers are specific to a user."

Reply:
Fixed both!